web/styles/dark_theme.css

```diff
@@ -1081,7 +1081,8 @@
 
     /* Search highlight used in both topics and rendered_markdown */
     .highlight {
-        background-color: hsl(51deg 100% 23%);
+        background-color: var(--color-background-search-highlight);
```

Comment:
The use of variables should mean striking all of these references from `dark_theme.css`. You need to make variables out of the light theme colors as well (I wondered why the dark mode variable stack had four variables that aren't declared on the light mode).